### PR TITLE
Error pages

### PR DESCRIPTION
**Pull request by itkg-ngilain**
https://github.com/open-orchestra/open-orchestra-cms-bundle/pull/961
https://github.com/open-orchestra/open-orchestra-model-interface/pull/98
https://github.com/open-orchestra/open-orchestra-model-bundle/pull/386
https://github.com/open-orchestra/open-orchestra-front-bundle/pull/111
https://github.com/open-orchestra/open-orchestra-front-demo/pull/67
https://github.com/open-orchestra/open-orchestra-provision/pull/15
https://github.com/open-orchestra/open-orchestra-docs/pull/66

### CODE REVIEW

File: deployment/tasks/error_pages.cap [hunk 1]
```diff
@@ -0,0 +1,9 @@
+namespace :orchestra do
+    desc "Generate error pages files"
+    task :generate_error_pages do
+        on roles(:app) do
+            execute :php, "#{fetch(:deploy_to)}/current/app/console orchestra:errorpages:generate --env=prod "
```
Comment (nicolasThal) [inline deployment/tasks/error_pages.cap]: as other command, would it be possible to run this command whitout running the deploy first ?

If yes, the `deploy_to` var is not available, I would advice you to put `~/current/` instead

Comment (itkg-ngilain) [inline deployment/tasks/error_pages.cap]: I'm not sure what you mean by is it "possible to run this command without running the deploy first" : That task is created to re-dump the error files after a deploy has occurred (maybe fixtures have changed).
Running that task without a deploy is useless

Comment (nicolasThal) [inline deployment/tasks/error_pages.cap]: when you do a deploy, the fixtures are not reloaded.

If you decide to reload the fixtures, then you could need to generate the page again

Comment (itkg-ngilain) [inline deployment/tasks/error_pages.cap]: Yes that's true I forgot about that. I switch to your proposition so

**Comment by itkg-ngilain** [comment]
@nicolasThal : is that update you wanted ?

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**